lib/systems/for-meta.nix

```diff
@@ -19,7 +19,11 @@ in rec {
   darwin  = [ patterns.isDarwin ];
   freebsd = [ patterns.isFreeBSD ];
   # Should be better, but MinGW is unclear, and HURD is bit-rotted.
-  gnu     = [ { kernel = parse.kernels.linux; abi = parse.abis.gnu; } ];
+  gnu     = [
+    { kernel = parse.kernels.linux; abi = parse.abis.gnu; }
+    { kernel = parse.kernels.linux; abi = parse.abis.gnueabi; }
+    { kernel = parse.kernels.linux; abi = parse.abis.gnueabihf; }
+  ];
```

Comment:
Just curious: why is this not just matching on ABI's? It seems unintuitive for "this package runs on GNU" to mean "gnu/linux" (and as a recent PR addressed seems to have been abused to mean "linux" in many packages).

But would it be "bad" to not require the kernel be linux?

Reply:
A good question. I was looking to minimize the potential risk incurred by my patch, but I'll admit that this definition is rather fishy.

Reply:
Haha, understood re:conservative changes.  I do the same thing all the time.  cc @Ericson2314 -- do you know the answer?

Not a big deal ... :)

Reply:
I'm fine either way. I made some changes upstream to GNU Config that are leading me to reconsider some of this stuff a bit. E.g. maybe ABI should be a per-kernel namespace.

 - http://lists.gnu.org/archive/html/config-patches/2018-04/threads.html
 - http://lists.gnu.org/archive/html/config-patches/2018-05/threads.html